### ISSUE TITLE
[Bug]:

### ISSUE THREAD

**Issue by H2M-GEM** [RedisInsight/RedisInsight#2562]
**Preconditions** (Any important steps we need to know)
when I use `redisInsight  cli`，I found
Why does the transaction have an error during exec that doesn't say what the redis website says it does

**Steps to reproduce** (How to reproduce what you found step by step)

```
127.0.0.1:6379[6]> multi
OK
127.0.0.1:6379[6]> set a abc
QUEUED
127.0.0.1:6379[6]> lpop a
QUEUED
127.0.0.1:6379[6]> set a abcd
QUEUED
127.0.0.1:6379[6]> exec
```

**Actual behavior** (A short description of what you found)
It confused me
![image](https://github.com/RedisInsight/RedisInsight/assets/41934045/ece1b854-c93b-449c-ae4b-2193a67df3bd)

**Expected behavior** (A short description of what you expected to find)

 result  like this :  exec returned three-element bulk string reply that there are two OK codes and the middle is an error reply
**Screenshots** (Paste or drag-and-drop a screenshot or a link to a recording)

when I use `redis-cli`，it does
![image](https://github.com/RedisInsight/RedisInsight/assets/41934045/7b545afa-dead-4e97-af77-2523c9cde65f)

**Additional context** (Operating system, version of RedisInsight, Redis database version, Redis module version, database type, connection type, logs, or any other information)

**Comment by ViktarStarastsenka** [comment]
hi @H2M-GEM , thank you for reporting this, it will be fixed in the upcoming releases.

**Comment by ViktarStarastsenka** [comment]
hi @H2M-GEM  this is fixed with 2.34 (released today). Please re-open the ticket if the issue is still relevant.
